src/main/game/CommandLine/CommandlineUi.java

```diff
@@ -103,6 +122,23 @@ public void askForLanguage() {
         output.println(english.askForLanguage());
     }
 
+    private String getTranslatedMove(String userMove) {
+        return translations.get(userMove);
+    }
+
+    private boolean languageIsEnglish() {
+        return language instanceof English;
+    }
+
+    private Moves getConvertedMove(String userMove) {
+        if (languageIsEnglish()) {
+            return convertMove(userMove);
+        } else {
+            String translatedMove = getTranslatedMove(userMove);
```

Comment:
How come we need to "work this hard" to get users move into a proper class `Moves`?
Also, why do we need bespoke methods if `Language` already understands that things might be English or Greek?